nestkernel/connection_manager.h

```diff
@@ -807,6 +815,12 @@ ConnectionManager::set_source_has_more_targets( const thread tid,
   connections_[ tid ][ syn_id ]->set_source_has_more_targets( lcid, more_targets );
 }
 
+inline void
+nest::ConnectionManager::set_has_get_connections_been_called( const bool has_get_connections_been_called )
```

Comment:
Should we instead have `set_...` and `unset_...` (without arguments) here for additional abstraction?

Reply:
I don't think that's necessary. It's just a bool being set to `true` or `false`. And this way we don't need to add an additional function to the already pretty crowded `ConnectionManager`.

Reply:
Alright. I was just wondering...